### DISCUSSION TITLE
jellyfin默认的帐号密码是多少

### DISCUSSION THREAD

**Discussion by SaulJWu** [opennaslab/kubespider#321] [Q&A] [answered]
按照官方教程，安装后，打开8096端口，发现需要账号密码阿
```bash
sudo bash hack/install_jellyfin.sh
```

没玩过，来个大佬带带我

**Comment by qingchoulove** [comment, marked as answer]
新安装的jellyfin是不需要密码的，大佬是否在安装时挂载了之前安装的配置目录？可以检查下 `${KUBESPIDER_HOME}/kubespider/jellyfin/`
这个路径，清空后重新执行安装应该就没有密码了
